src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/deciders/configs/DeciderConfig.java

```diff
@@ -26,7 +24,8 @@
  *     // Decreasing order of priority for the type of workload we can expect on the cluster.
  *     // Priority order in the list goes from most expected to the lease expected workload type.
  *     "workload-type": {
- *       "priority-order": ["ingest", "search"]
+ *       "prefer-ingest": true,
+ *       "prefer-search": false
```

Comment:
What happens when there are more than 2 workload type. If we dont want them in a list, can we use numbers to denote their priority? Also can you update the comment if we are changing it

Reply:
I actually had a discussion with Vigya on this in my previous PR. The problem with setting workload as a list is we have to maintain the string parsing logic inside the Config object and the priority list does not allow customer to specify the scenario if both search/ingest are equally important to customer or their simply don't have any preference. 
So we created two options here instead so that customer can explicitly specific whether search/ingest is preferable. 
The comments has been updated in the WorkLoaderConfig.java